distributed/tests/test_worker.py

```diff
@@ -1356,9 +1356,9 @@ async def test_interface_async(cleanup, loop, Worker):
 
 
 @pytest.mark.gpu
-@pytest.mark.asyncio
 @pytest.mark.parametrize("Worker", [Worker, Nanny])
-async def test_protocol_from_scheduler_address(cleanup, Worker):
+@gen_test()
+async def test_protocol_from_scheduler_address(Worker):
```

Comment:
cleanup used to call `gc.collect()`. Is there a particular reason this fixture isn't necessary any more?

Reply:
cleanup adds `with clean:` which is already in `gen_test`
https://github.com/dask/distributed/blob/26ce9bbf4f2ab2faf632f7763bab4b8c81297acb/distributed/utils_test.py#L1767

Reply:
the cleanup is part of gen_test. cleanup just calls a contextmanager that is also invoked by gen_test